tests/test_preprocess.py

```diff
@@ -124,3 +125,33 @@ def test_extend_lines(self):
         ext5 = mm.extend_lines(gdf, 2)
         assert ext5.length.sum() > gdf.length.sum()
         assert ext5.length.sum() == pytest.approx(6.2, rel=1e-3)
+
+    def test_roundabout_simplification(self):
```

Comment:
Would it make more sense to break this one test of 5 checks into 5 tests? Something as simple as `test_roundabout_simplification_1()`,   `test_roundabout_simplification_2()`, etc.

The data reads an polygonize can still be preformed once in the `setup_method()`.

Reply:
That makes sense... I'll implement that, thanks !